crates/udd-sketch/src/lib.rs

```diff
@@ -285,6 +290,51 @@ impl UDDSketch {
         self.values_sum += value;
     }
 
+    /// `merge_items` will merge these values into the current sketch
+    /// it requires less memory than `merge_sketch`, as that needs a fully serialized
+    /// `UDDSketch`, whereas this function relies on iterators to do its job.
+    pub fn merge_items(
```

Comment:
unit Tests are missing. Thinking of running all the tests we currently have bothe against `merge_items` and `merge_sketches`